lib/winston-utils.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+exports.parseProtocol = function parseProtocol(protocol) {
```

Comment:
Für diese eine Funktion, die nur an einer Stelle verwendet wird, brauchen wir keine extra Datei. Bitte in `winston-syslog.js` integrieren.